suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -1036,3 +1036,9 @@ export const selectDeviceUpdateFirmwareVersion = (state: DeviceRootState) => {
 
     return device ? getFwUpdateVersion(device) : null;
 };
+
+export const selectFirmwareChangelog = (state: DeviceRootState) => {
+    const device = selectSelectedDevice(state);
+
+    return device?.firmwareRelease?.changelog?.[0]?.changelog;
```

Comment:
We should definitely take `changelog_bitcoinonly` into account for BTC-only devices.